<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>invoice</class>
 <widget class="QDialog" name="invoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="customerComboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="branchComboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="methodCombobox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Branch</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Method</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>58</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>YYYY-MM-DD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
